internal/ingress/types.go

```diff
@@ -48,6 +48,7 @@ var (
 // Configuration holds the definition of all the parts required to describe all
 // ingresses reachable by the ingress controller (using a filter by namespace)
 type Configuration struct {
+
```

Comment:
is this Go convention to have an extra line here?

Reply:
I don't remember adding this line, maybe gofmt did? I'll remove it and see if any checks yell at me

Reply:
Nice. I would use goimports, but CI for ingress-nginx requires gofmt to be run